config/webpack.config.dev.js

```diff
@@ -0,0 +1,16 @@
+const path = require("path");
+const config = require("./webpack.config.js");
+
+config.devServer = {
+  historyApiFallback: true, //serve a previous page on a 404 error
+  contentBase: path.resolve("src"), // location of the source code
+  port: 8080, // use this port for the server
+  hot: true, // refresh the browser when changes are saved
+  open: true, // open the project in the browser when the server starts
+  host: "0.0.0.0", // make server accessible externally
```

Comment:
`host: "0.0.0.0", // make server accessible externally` This works on mac but not on windows - something to take into consideration :)